lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +257,68 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
+        self._mystem = Mystem()
+        tag_mapping_path = Path(__file__).parent / 'data' / 'mystem_tags_mapping.json'
+        self._tag_converter = MystemTagConverter(tag_mapping_path)
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        conllu_sentences = []
+        result = (i for i in self._mystem.analyze(text))
+
+        for idx_sent, sentence in enumerate(split_by_sentence(text)):
+            conllu_tokens = []
+            tokens = []
+
+            for token in result:
+                if token['text'] not in sentence:
```

Comment:
can you explain in which cases this condition is true?

Reply:
it is true when the word (from analyzed by mystem text) is not in current sentence